src/js/jagex2/config/FloType.ts

```diff
@@ -94,7 +94,7 @@ export default class FloType extends ConfigType {
         } else if (code === 6) {
             this.debugname = dat.gjstr;
         } else {
-            console.log('Error unrecognised config code: ', code);
+            throw new Error(`Unrecognised config code: ${code}`);
```

Comment:
Looking at the Java client for example:
`https://github.com/2004Scape/Client/blob/main/runetek3/src/main/java/jagex2/config/FloType.java#L91`

Does not throw an error.